tests/frontier/opcodes/test_calldatasize.py

```diff
@@ -0,0 +1,98 @@
+"""test `CALLDATASIZE` opcode."""
+
+import pytest
+
+from ethereum_test_forks import Byzantium, Fork
+from ethereum_test_tools import Account, Alloc, StateTestFiller, Transaction
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+
+@pytest.mark.parametrize(
+    "args_size,sha3_len,storage",
+    [
+        ("0002", "01", Account(storage={0x00: 0x02})),
```

Comment:
```suggestion
        ("0002", "01", 0x02),
```
Similar comment to previous file, `Account(storage=...` can be defined inside the function to make the parametrization more readable.